### ISSUE TITLE
[P2][Sourcing][Accessibility][Pretest][Multi input autocomplete] When navigating into the Autocomplete list with the arrow keys, the focus is at the top of the page when TAB is pressed. 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10816]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.

When navigating into the Autocomplete list with the arrow keys, the focus is at the top of the page when TAB is pressed. 
Keyboard focus should be placed at the next control.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
[Screen Recording 2023-10-26 at 4.29.12 PM.mov.zip](https://github.com/SAP/fundamental-ngx/files/13176479/Screen.Recording.2023-10-26.at.4.29.12.PM.mov.zip)

press tab 2 times atfter pressing up or down arrow while in the list

**Comment by N1XUS** [comment]
Fixed with #10853
